scripts/setup/verify_dev_environment.py

```
@@ -0,0 +1,456 @@
+            name="PostgreSQL",
+            passed=False,
+            message="DATABASE_URL not set in environment",
+        )
+
+    try:
+        conn = await asyncpg.connect(db_url, timeout=5)
+        version = await conn.fetchval("SELECT version()")
+        await conn.close()
+
+        return HealthCheck(name="PostgreSQL", passed=True, message=f"Connected: {version[:50]}...")
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)